docs/getting-started/integrating-codacy-with-visual-studio-code.md

```diff
@@ -19,46 +79,29 @@ Use this extension to get the full list of problems found by Codacy for a pull r
 
 ## Getting pull request quality and coverage data
 
-To see Codacy quality and coverage data for the pull request you're working on, follow these steps:
+To see Codacy quality and coverage data for a pull request, follow these steps:
```

Comment:
When we had "the pull request you're working on", it was implicit they were open. Removing that expression, it's not so clear anymore... 
Again for the sake of technical accuracy, I would keep "**open** pull request". And also because it's short, it doesn't increase the sentence complexity, IMO.